src/Order/OrderResource.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace ShoppingFeed\Sdk\Order;
+
+use ShoppingFeed\Sdk\Api\Store\StoreResource;
+use ShoppingFeed\Sdk\Resource\AbstractResource;
+
+class OrderResource extends AbstractResource
+{
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return (int) $this->getProperty('id');
+    }
+
+    /**
+     * @return int
+     */
+    public function getStoreId()
+    {
+        return (int) $this->getProperty('storeId');
+    }
+
+    /**
+     * @return string
+     */
+    public function getReference()
+    {
+        return (string) $this->getProperty('reference');
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getStoreReference()
+    {
+        return $this->getProperty('storeReference');
+    }
+
+    /**
+     * @return null|\DateTimeImmutable
+     */
+    public function getAcknowledgedAt()
+    {
+        return date_create_immutable($this->getProperty('acknowledgedAt'));
```

Comment:
The acknowledgement date should be checked before to creates a date : in case of NULL value, a date object will be created with the current timestamp. Also, a failed `date_create_immutable` call will return false, which does not fit the documentation